sparkmonitoring/api.py

```diff
@@ -27,6 +27,15 @@ def _do_request(self, path, params=None):
 
         return r.json()
 
+    def _do_request_file(self, path, params=None):
+        url = self._base_url + path
+        r = requests.get(url, params)
+
+        if r.status_code != requests.codes.ok:
+            r.raise_for_status()
+
+        return r.content
```

Comment:
Do you think it's worth keeping this as a single method, and changing the response type?

Eg, 

```
def _do_request(self, path, params=None, response_type=ResponseType.JSON):
   ....
   if response_type == ResponseType.JSON:
        return r.json()
   else:
       return r.content

```